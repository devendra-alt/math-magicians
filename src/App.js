import Calculator from './components/calculator';
import './styles/App.css';

function App() {
  return <Calculator />;
}

export default App;
