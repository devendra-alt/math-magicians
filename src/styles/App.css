/* calculator */
:root {
  --calculator-screen-primary: #e0e0e0;
  --calculator-screen-ans-primary: #858694;
  --calculator-screen-last-row: #f5913e;
  --primary-white: #fff;
  --primary-black: #000;
}

.calculator-container {
  display: flex;
}

table {
  margin: 0 auto;
}

td {
  width: 10rem;
  height: 5.5rem;
  text-align: center;
  background-color: var(--calculator-screen-primary);
  font-size: 1.4rem;
  font-weight: 500;
  cursor: pointer;
}

td:last-child {
  background-color: var(--calculator-screen-last-row);
}

#calculator-answer-el {
  background-color: var(--calculator-screen-ans-primary);
  text-align: right;
  padding-right: 2.5rem;
  font-size: 2rem;
  font-weight: 700;
  color: var(--primary-white);
  cursor: auto;
}

::selection {
  color: #000;
}

@media (min-width: 764px) {
  .calculator-container {
    justify-content: flex-end;
    padding: 8rem 16rem;
  }

  table {
    margin: 0;
  }
}
