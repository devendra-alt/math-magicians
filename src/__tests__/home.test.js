import renderer from 'react-test-renderer';
import App from '../App.js';
describe('renders correctly', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
