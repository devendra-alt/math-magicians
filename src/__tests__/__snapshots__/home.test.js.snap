// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly renders correctly 1`] = `
Array [
  <header
    className="header"
  >
    <nav
      className="nav-bar"
    >
      <h1
        className="title"
      >
        Math Magicians
      </h1>
      <ul
        className="nav-links"
      >
        <li
          className="nav-link-item"
        >
          <a
            className="nav-link"
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li
          className="nav-link-item"
        >
          <a
            className="nav-link"
            href="/calculator"
            onClick={[Function]}
          >
            Calculator
          </a>
        </li>
        <li
          className="nav-link-item"
        >
          <a
            className="nav-link"
            href="/quotes"
            onClick={[Function]}
          >
            Quotes
          </a>
        </li>
      </ul>
    </nav>
  </header>,
  <main
    className="main-landing-page"
  >
    <h1
      className="main-landing-page-title"
    >
      Welcome to our page
    </h1>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia, molestiae quas vel sint commodi repudiandae consequuntur voluptatum laborum numquam blanditiis harum quisquam eius sed odit fugiat iusto fuga praesentium optio, eaque rerum! Provident similique accusantium nemo autem. Veritatis obcaecati tenetur iure eius earum ut molestias architecto voluptate aliquam nihil, eveniet aliquid culpa officia aut! Impedit sit sunt quaerat, odit, tenetur error, harum nesciunt ipsum debitis quas aliquid. Reprehenderit, quia. Quo neque error repudiandae fuga? Ipsa laudantium molestias eos sapiente officiis modi at sunt excepturi expedita sint? Sed quibusdam recusandae alias error harum maxime adipisci amet laborum. Perspiciatis minima nesciunt dolorem! Officiis iure rerum voluptates a cumque velit quibusdam sed amet tempora. Sit laborum ab, eius fugit doloribus tenetur fugiat, temporibus enim commodi iusto libero magni deleniti quod quam consequuntur! Commodi minima excepturi repudiandae velit hic maxime doloremque. Quaerat provident commodi consectetur veniam similique ad earum omnis ipsum saepe, voluptas, hic voluptates pariatur est explicabo fugiat, dolorum eligendi quam cupiditate excepturi mollitia maiores labore suscipit quas? Nulla, placeat. Voluptatem quaerat non architecto ab laudantium modi minima sunt esse temporibus sint culpa, recusandae aliquam numquam totam ratione voluptas quod exercitationem fuga. Possimus quis earum veniam quasi aliquam eligendi, placeat qui corporis!
    </p>
  </main>,
]
`;
